docs/gaia/launch/blog-1-sp.md

```diff
@@ -0,0 +1,181 @@
+tercera auditoría.
+
+### 4 Cosmos SDK Suspensión de Funciones
+
+Los cambios finales en la ruptura del Cosmos-SDK están incluidos en el [v0.31.0 launch RC](https://github.com/cosmos/cosmos-sdk/projects/27). 
+Una vez que se complete este RC, el equipo de Cosmos-SDK participará en una ronda de búsqueda interna de errores para garantizar 
+una diligencia debida de seguridad suficiente previa al lanzamiento.
+
+Inmediatamente después del lanzamiento de Cosmos-SDK v0.31.0, se lanzará una red de pruebas de Gaia en un esfuerzo por 
+eliminar cualquier error difícil de encontrar.
+
+### 3 Game of Stakes Completado
+
+Game of Stakes (GoS),[el primer concurso de redes de prueba adversarias de su tipo](https://blog.cosmos.network/announcing-incentivized-testnet-game-efe64e0956f6), 
+se lanzó en diciembre de 2018 para poner a prueba los incentivos económicos y los estratos sociales de una red de cadenas de bloques asegurada exclusivamente 
+por la prueba de participación. Hasta la fecha, la cadena de bloques de GoS ha tenido hard-forks con éxito en tres ocasiones.
+Tan pronto como el módulo GoS concluya, se utilizarán los [criterios de puntuación](https://github.com/cosmos/game-of-stakes/blob/master/README.md#scoring) para determinar 
+los ganadores. Estos serán anunciados después de la finalización del juego.
+
+### 2 Transacciones para el Genesis recopiladas
+
+La Fundación Interchain publicará una recomendación para la asignación de Atoms en el génesis. Esto incluirá asignaciones 
+para los participantes de la recaudación de fondos de Cosmos, los primeros contribuyentes y los ganadores del Game of Stakes. 
+Cualquier persona con una asignación recomendada tendrá la oportunidad de presentar un gentx, el cual es requerido para 
+convertirse en validador de genesis. El resultado final de la asignación recomendada y la colección de gentxs es 
+un [archivo de génesis](https://forum.cosmos.network/t/genesis-files-network-starts-vs-upgrades/1464) final.
+
+### 1 Lanzamiento del Cosmos Hub en Mainnet
+
+Una vez que un archivo de génesis es aprobado por la comunidad, y +⅔ del poder de voto se pone en línea, la red principal 
+de Cosmos estará en vivo.
+
+## Canales Oficiales de comunicacion de Cosmos
+
+Estas son las cuentas oficiales que comunicarán los detalles del lanzamiento:
+
+- [Cosmos Network](https://twitter.com/cosmos)
+- [Cosmos GitHub](https://github.com/cosmos)
+- [Cosmos Blog](https://blog.cosmos.network)
+
+Por favor, tenga en cuenta que el [foro](https://forum.cosmos.network/),[grupos de chat de RIOT](https://riot.im/app/#/group/+cosmos:matrix.org), y [grupo de Telegram](http://t.me/cosmosproject)
+no deben ser tratados como noticias oficiales de Cosmos.
+
+Si tiene dudas o confusión sobre los próximos pasos a seguir y no está seguro sobre fuentes de información confiables, 
+no haga nada durante el período inicial y espere una actualización a través de los tres canales de comunicación mencionados 
```

Comment:
Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.